daemon/logger/jsonfilelog/multireader/multireader.go

```diff
@@ -67,7 +69,9 @@ func (r *multiReadSeeker) Seek(offset int64, whence int) (int64, error) {
 			}
 			tmpOffset += s
 		}
-		r.Seek(tmpOffset+offset, os.SEEK_SET)
+		if _, err := r.Seek(tmpOffset+offset, os.SEEK_SET); err != nil {
```

Comment:
Rather than calling `Seek` recursively here, which seems to do a lot of redundant work, couldn't we just set `r.pos` directly?